directories-list.md

```diff
@@ -0,0 +1,157 @@
+
+- [orbit-db-cache](https://github.com/orbitdb/orbit-db-cache)
+
+Module that orbitdb uses. JS, npm. Actually a persistency layer, not a cache. Saves the latest HEAD of each db locally into levelDB. Doesn't store a huge amount of data, just the amount of DBs you have. For all of the dbs you have locally, you have a hash for each.
+
+- [orbit-db-cli](https://github.com/orbitdb/orbit-db-cli)
+
+CLI for orbit-db.
+
+- [orbit-db-counterstore](https://github.com/orbitdb/orbit-db-counterstore)
+
+Datastore in orbitdb. Uses CRDTs, inherits from orbit-db-store. Why is it called counterstore and not CRDTstore? Because it is only a counter - it uses CRDTs for that. Only for gcounter, more specifically.
+
+- [orbit-db-docstore](https://github.com/orbitdb/orbit-db-docstore)
+
+Another orbit-db store which allows you to index a blob. npm js.
+
+- [orbit-db-eventstore](https://github.com/orbitdb/orbit-db-eventstore)
+
+Another data model type. Like IPFS log. Wraps into a store interface.
+
+- [orbit-db-feedstore](https://github.com/orbitdb/orbit-db-feedstore)
+
+Database type in orbitdb, exactly like eventlog, but you can delete events from the log. Twitter feed you can delete your entries - this is the same.
+
+- [orbit-db-keystore](https://github.com/orbitdb/orbit-db-keystore)
+
+Keystore is actually a module that stores the keys locally. You can generate keys, it persists them in the data folder, you can import and export keys - it is basically a key manager.
+
+- [orbit-db-kvstore](https://github.com/orbitdb/orbit-db-kvstore)
+
+KeyValue store. Another DB type in OrbitDB. May be unclear in the code, where `kvstore` is called `keyvalue` store.
+
+- [orbit-db-pubsub](https://github.com/orbitdb/orbit-db-pubsub)
+
+A layer between orbitdb and IPFS Pubsub. Basically is the message broker of Orbitdb that handles all fo the communication. Distributed pubsub. Every update goes through this. Every orbitdb address is a channel in pubsub, and when you open the database you join that pubsub channel.
+
+- [orbit-db-store](https://github.com/orbitdb/orbit-db-store)
+
+This is a base calls for all datastores in orbitdb. A Base Class defines the common interface for all databases. We should refactor this - it should not be a base class, but a composting (?). We'll figure this out. The Stores are the API of that database; they also contian the index so that if you cac he anything in the memory, the user never actually directly asks the index, but the API of the store. All of these stores use -- they're very simple, there's no io or so on. The store is where that happens. Every store calls this operation which calls the log, and so on, and eventually sends a cb somewhere saying that it is persisted. It has a load method that checks the cache, then ... I lost it. The base class hooks the orbitdb main thing, that contains the messaging and the keystore and the access-controller with the log and the form and so on with the indices.
```

Comment:
s/calls/class
s/contian/contain
s/cac he/cache

Reply:
I think this was for an older commit? :)